usr/src/uts/common/inet/ip/spd.c

```diff
@@ -5376,7 +5377,7 @@ ipsec_tun_outbound(mblk_t *mp, iptun_t *iptun, ipha_t *inner_ipv4,
 		 * NOTE2:  "negotiate transport" tunnels should match ALL
 		 * inbound packets, but we do not uncomment the ASSERT()
 		 * below because if/when we open PF_POLICY, a user can
-		 * shoot him/her-self in the foot with a 0 priority.
+		 * shoot themself in the foot with a 0 priority.
```

Comment:
Aw shoot, I thought I got it right with him/her (yes, I wrote this line).  Oh well.